flutter/rsvp/lib/Models/LocalData.dart

```diff
@@ -29,8 +34,21 @@ class LocalData implements IlocalData {
     return null;
   }
 
-  List<String> loadEvents() {
```

Comment:
1. I think loadEvents (or other methods like `getCreatedEvents`) should return a list of Events not a list of Strings.
2. Didn't we want to have seperate lists for created events and joined events in the storage, but have one list in view to display all of them on one page?  ViewModel knows that view needs to display both created events and joined events in the same list. However, we decided to have two lists in the local storage, indeed two methods to retrieve them.
3. We already have `getCreatedEvents` and `getRespondedEvents` in the `LocalData` class. Does this loading method has different functionality?